tests/fixtures/findings-to-issues/disposition.phase2.yaml

```
@@ -0,0 +1,2 @@
+findings_disposition:
+  - { file: "src/dup.ts", line: 3, disposition: APPLIED }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)